src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let counter = 0;
+
+  const adder = (num) => {
+    if (num === undefined) {
```

Comment:
Checking for `undefined` using `num === undefined` might not be the best approach if the user explicitly passes `undefined` as an argument. Consider using `arguments.length` to check if any arguments were passed instead.